app/src/main/java/com/example/whattowatch/adapters/RecommenderAdapter.java

```diff
@@ -0,0 +1,68 @@
+package com.example.whattowatch.adapters;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.bumptech.glide.Glide;
+import com.example.whattowatch.R;
+import com.example.whattowatch.models.Movie;
+
+import java.util.List;
+
+public class RecommenderAdapter extends RecyclerView.Adapter<RecommenderAdapter.ViewHolder>{
+    public static final String TAG = "RecommenderAdapter";
+
+    private Context context;
+    private List<Movie> movies;
+
+    public RecommenderAdapter(Context context, List<Movie> movies) {
+        this.context = context;
+        this.movies = movies;
+    }
+
+    @NonNull
```

Comment:
curious, what is this decorator for?

Reply:
@NonNull basically checks to make sure that a certain parameter is not null, in this case we have to make sure that parent parameter is a true instance of ViewGroup and is not just null, so we have NonNull to do check